add warning about configuring aws credentials for Kubernetes deployments (#15374)
pulumi/docs: content/docs/iac/get-started/kubernetes/deploy-stack.md

{{< console-note >}}

> <div class="note note-info">
>     <div class="icon-and-line">
>         <i class="fas fa-info-circle"></i>
>         <div class="line"></div>
>     </div>
>     <div class="content">
>        If you get the error <code>configured Kubernetes cluster is unreachable: unable to load schema information from the API server: the server has asked for the client to provide credentials</code>, you may need to configure valid AWS credentials.
>     </div>
> </div>

Reviewer: ```suggestion
{{% notes type="info" %}}
If you get the error `configured Kubernetes cluster is unreachable: unable to load schema information from the API server: the server has asked for the client to provide credentials`, you may need to configure valid AWS credentials.
{{% /notes %}}
```
